cloudinit/cmd/main.py

```diff
@@ -72,23 +72,32 @@ def print_exc(msg=""):
     sys.stderr.write("\n")
 
 
+DEPRECATE_BOOT_STAGE_MESSAGE = (
+    "Triggering cloud-init boot stages outside of intial system boot is not a"
+    " fully supported operation which can lead to incomplete of incorrect"
+    " configuration due to either race-conditions with other active services"
+    " or incomplete triggering of all cloud-init boot stages. As such,"
+    " cloud-init is deprecating this undocumented feature in the future. If"
+    " there are use-cases in an environment which requires calling cloud-init"
+    " boot stages sometime after boot, please file an issue to allow"
```

Comment:
Lets tell the users what we are looking for, exactly, from this request to file an issue.


Something like this maybe?

"If you currently use cloud-init in this way, please file an issue describing in detail your use case so that cloud-init can better support your needs."